client/my-sites/customize/main.jsx

```diff
@@ -281,14 +283,19 @@ class Customize extends Component {
 					}
 					break;
 				case 'activated':
+					debug( 'iframe says it activated a theme', message );
```

Comment:
Are we going to remove these debugging messages?

Reply:
No. We should have more debugging message in our code not less so it's easier to understand how the app is behind in production (the checkout code is full of them and it lets them diagnose problems much faster. 

The debug messages are only activated if you do the `localStorage.setItem( 'debug', 'calypso:my-sites:customize' );` thing anyway

Reply:
if you look at the rest of this method, it already has debug messages for the other cases.